app/src/main/java/com/bignerdranch/android/criminalintent/controllers/CrimeFragment.kt

```diff
@@ -44,9 +42,9 @@ class CrimeFragment : Fragment(R.layout.fragment_crime) {
         super.onStart()
 
         val titleWatcher = object : TextWatcher {
-            override fun afterTextChanged(s: Editable?) {}
+            override fun afterTextChanged(s: Editable?) = Unit
```

Comment:
why? I think the previous way is better. @FerrariBruno @f7deleon thought ?

Reply:
Detekt did not like the previous option

Reply:
maybe add a space between the brackets? that may be it